### DISCUSSION TITLE
Impossibile trovare device su Zigbee2MQTT utilizzando Conbee II

### DISCUSSION THREAD

**Discussion by dragone125** [indomus/forum#3176] [ZigBee, Z-Wave, Thread, Matter]
Buongiorno a tutti, sono giunto qui, dopo averne parlato anche con @KC-inDomus sulla chat telegram, poiché è da tempo che ho un annoso problema con Zigbee2MQTT. Ho seguito la guida di inDomus.it e sono riuscito a instanziare Z2M correttamente come docker su RaspberryPi OS. Riesco ad aprire il frontend e a mappare la mia chiavetta CONBEE II ma non riesco in nessun modo e con nessuna versione ad aggiungere (più) i dispositivi zigbee. il "più" tra parentesi è dovuto al fatto che in una precedente installazione tramite HassOS, riuscivo ad usare correttamente l'add on Z2M

Qualcuno potrebbe provare ad aiutarmi? Posso fornire log e screenshot se necessario

Grazie a tutti

**Comment by dragone125** [comment]
Buonasera, provo a riportare questo problema in alto perché ancora non riesco a risolvere da quasi un anno.

**Reply by KC-inDomus** [reply]
ciao, sinceramente non capisco/visualizzo di preciso il problema. hai provato a  fermare l'istanza, crearne un'altra e vedere se la nuova funziona? il firmware di conbee è aggiornato?

**Reply by dragone125** [reply]
Ciao @KC-inDomus, ho provato a cancellare l’istanza, fermarla, ho addirittura provato a impostare home assistant da zero passando da un’installazione core a una supervised ma niente di tutto questo ha funzionato. Il coordinatore è un ConbeeII con firmware 0x26720700, che viene riconosciuto in Z2M. Il problema è che io posseggo dispositivi IKEA, Aqaara e BTicino che comunicano via Zigbee ma non riesco ad associare nessuno di questi.

**Reply by KC-inDomus** [reply]
ma esattamente che ti dice sul log, quando provi a effettuare il pairing?

**Reply by dragone125** [reply]
> ma esattamente che ti dice sul log, quando provi a effettuare il pairing?

2024-09-04 15:29:16z2m: Zigbee: allowing new devices to join.
Info 2024-09-04 15:29:16z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/bridge/response/permit_join', payload '{"data":{"time":254,"value":true},"status":"ok","transaction":"9rw1w-1"}'

**Reply by KC-inDomus** [reply]
> 2024-09-04 15:29:16z2m: Zigbee: allowing new devices to join.
> 
> Info 2024-09-04 15:29:16z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/bridge/response/permit_join', payload '{"data":{"time":254,"value":true},"status":"ok","transaction":"9rw1w-1"}'

questo determina che il bridge si è messo in ascolto per il paring. ma quando "fallisce" ad associare? lascia traccia sul log?

**Reply by dragone125** [reply]
Purtroppo nessuna traccia. Il dispositivo lampeggia come da descrizione del pairing ma non avviene assolutamente niente lato Z2M

> > 2024-09-04 15:29:16z2m: Zigbee: allowing new devices to join.
> > Info 2024-09-04 15:29:16z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/bridge/response/permit_join', payload '{"data":{"time":254,"value":true},"status":"ok","transaction":"9rw1w-1"}'
> 
> questo determina che il bridge si è messo in ascolto per il paring. ma quando "fallisce" ad associare? lascia traccia sul log?

**Reply by snakuzzo** [reply]
Puoi condividere la configurazione di Z2M ?
Altra domanda...hai per caso provato a fare la stessa cosa con ZHA al posto di Z2M ?

**Reply by dragone125** [reply]
> Puoi condividere la configurazione di Z2M ? Altra domanda...hai per caso provato a fare la stessa cosa con ZHA al posto di Z2M ?

Ciao @snakuzzo, sotto trovi la configurazione di Z2M. Ho oscurato solo le credenziali di mqtt e l'indirizzo del server ma riesco a raggiungere il broker. sì, ho provato anche ZHA ma l'ho dovuto cancellare per conflitto tra i container. ad ogni modo il risultato era lo stesso, il ConBeeII viene riconosciuto e riesco a visualizzare le info ma non c'è modo di eseguire il pairing dei device zigbee.

homeassistant: true
permit_join: true
mqtt:
  base_topic: zigbee2mqtt
  server: mqtt://192.168.XX.XX
  user: *********
  password: **********
  version: 5
  include_device_information: true
  keepalive: 60
  reject_unauthorized: true
serial:
  port: /dev/ttyACM0
  adapter: deconz
advanced:
  log_directory: /var/log
  log_syslog:
    app_name: Zigbee2MQTT
    eol: /n
    host: localhost
    localhost: localhost
    path: /dev/log
    pid: process.pid
    port: 514
    protocol: udp4
    type: '5424'
  channel: 20
  log_level: info
experimental: {}
frontend:
  port: 8080
ota:
  update_check_interval: 10
disable_automatic_update_check: false
devices: {}

**Reply by snakuzzo** [reply]
Al netto dell'indentazione, visto che non hai usato il markdown, mi sembra ok.
Visto che mi pare tu non abbia alcun device già configurato, hai provato a forzare un canale diverso?

**Reply by dragone125** [reply]
> Al netto dell'indentazione, visto che non hai usato il markdown, mi sembra ok. Visto che mi pare tu non abbia alcun device già configurato, hai provato a forzare un canale diverso?

sì ci ho provato un bel po' di tempo fa ma onestamente sono andato un po' a caso sui canali

**Reply by snakuzzo** [reply]
Hai provato a collegare la chiavetta con una prolunga e non direttamente alla porta USB del server?

**Reply by dragone125** [reply]
> Hai provato a collegare la chiavetta con una prolunga e non direttamente alla porta USB del server?

ho provato a cambiare porta. uso un raspberry PI 4 e ho 4 porte USB di cui 3 libere (1 è dedicata all'SSD su cui è installato HA e tutti i container). non ho provato ad usare una prolunga in mezzo tra il raspberry e la chiavetta, ma non sarebbe peggio? o intendi altro?

**Reply by snakuzzo** [reply]
No, anzi... è consigliato l'uso di una prolunga USB per evitare interferenze.
Prova inoltre e su una porta USB2 e non USB3

**Reply by dragone125** [reply]
> No, anzi... è consigliato l'uso di una prolunga USB per evitare interferenze. Prova inoltre e su una porta USB2 e non USB3

Grazie, provo a recuperare una prolunga e vi aggiorno.

**Reply by KC-inDomus** [reply]
> No, anzi... è consigliato l'uso di una prolunga USB per evitare interferenze. Prova inoltre e su una porta USB2 e non USB3

non ci avevo pensato, bravo. è possibilissimo sia quello

**Reply by dragone125** [reply]
> > No, anzi... è consigliato l'uso di una prolunga USB per evitare interferenze. Prova inoltre e su una porta USB2 e non USB3
> 
> Grazie, provo a recuperare una prolunga e vi aggiorno.

Buongiorno a tutti,
ho fatto la prova richiesta da @snakuzzo facendomi prestare una prolunga USB2.0 da ben 5 m. ho collegato la prolunga al Raspberry che ospita HomeAssistant e il Conbee2 dalla parte opposta. Ho provato a fare il pairing del mio sensore porta Aqara per l'ennesima volta ma questa volta il pairing è avvenuto con successo al primo colpo, con mia enorme sorpresa.
grazie per il supporto che mi avete fornito. ora però vi chiedo, è possibile impostare qualche settaggio così da poter collegare il dongle direttamente al raspberry? la prolunga in questo modo non posso tenerla.

**Reply by Scialla** [reply]
non ci sono settaggi, devi allontanare il dongle dal rasp perché le frequenze interferiscono.. Non è necessario usare una prolunga da 5 mt, basta un cavo di qualche decina di cm
